src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_panel.scss

```diff
@@ -4,6 +4,10 @@
   * .embPanel--editing doesn't get updating without a hard refresh
   */
 
+.dshDashboardGrid__item {
+  scroll-margin-top: calc(var(--euiFixedHeadersOffset, 100) * 4);
```

Comment:
Can you explain why you chose this number? I think it's too large, so I'm not getting the best behaviour 🤷

Reply:
Hmmm... how do you feel about adding the scrolling back and then removing this? 🤔 Otherwise the top of the chart is still not visible to the user.

```css
.coreSystemRootDomElement {
  overflow-y: auto !important;
}
```

Reply:
I still don't like this, no - it goes against what the goal of https://github.com/elastic/kibana/issues/185895 was.

> Otherwise the top of the chart is still not visible to the user. 

Did you try making the `scroll-margin-top` smaller?

Reply:
![Screenshot 2024-10-04 at 9 33 48 AM](https://github.com/user-attachments/assets/f0e3b6a8-18a9-4e02-a401-8eeb7d37443a)
This is with 194 px as the scroll-margin-top property - does this work on in your testing? Any smaller than this and without the ability to scroll, I think wouldn't be a good solution since then users can't see the top of the chart still. 


```css
.dshDashboardGrid__item {
  scroll-margin-top: calc(var(--euiFixedHeadersOffset, 100)* 2);
}
```

Reply:
Committed with changes in [2b7b8da](https://github.com/elastic/kibana/pull/193430/commits/2b7b8dacb934954d331169a33ef4c5308f8def35) let me know what you think thanks!!